src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,57 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp(){
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("set size의 메소드 활용하여 크기를 확인")
+    void set_requireStep1(){
```

Comment:
```suggestion
    void set_requireStep1() {
```

앞으로의 미션에서 자바 컨벤션을 지켜주시면 좋을 것 같습니다!